chainer/link.py

```diff
@@ -558,6 +558,28 @@ def serialize(self, serializer):
         for name in self._persistent:
             d[name] = serializer(name, d[name])
 
+    @property
+    def size(self):
+        """Count the size of parameters.
```

Comment:
I thought "number of parameters" could mean the number of "Parameter" objects in the link. So, I used "size of parameters". How do you think about it?